### ISSUE TITLE
[Question] Name and branding

### ISSUE THREAD

**Issue by myfonj** [rxi/lite#156]
tl;dr: literally "lite" & icon/logo from icon.ico?

I'm really impressed by this project an I'd like to see this editor listed in alternativeto.net database (and similar). However I'm not sure about current application name, "lite" *per se* sounds very generic, vague, hard to search for, and I wouldn't be surprised if it even caused some copyright trouble in the future (IANAL).  

So before submitting anywhere I'd like to make sure that:
- **Should it be listed as "`lite`"** or should it rather be called by some more unique name, like `rxi lite` / `lite by rxi` / `rxi-lite`  / `lite editor` **or other**?
- Initial `l` is really supposed to be **lowercase**, right?
- **/icon.ico** can be used as "Product icon / logo" verbatim (or up-scaled to multiple without aliasing)?

---

Bonus question: is author related to creator of https://alternativeto.net/software/sfxia/ ?

**Comment by rxi** [comment]
Thanks for the kind words!

* I've seen people refer to it as `rxi/lite` (I assume in reference to the URL) to avoid ambiguity, this seems like a good choice if `lite` is too generic a name for your use case
* The `l` is typically lowercase
* Someone went to the effort of making an [svg version of the icon](https://github.com/rxi/lite/issues/134) if you need it in a larger size 

---

Bonus answer: `sfxia` was a small weekend project I made *(not sure why its listed there as windows only despite also having a linux version)*
